bin/install.sh

```diff
@@ -200,10 +197,6 @@ function run_ansible_playbook() {
 function upgrade_docker_containers() {
     display_section "Initialize/Upgrade Docker Containers"
 
-    wget -q \
```

Comment:
Why was this removed?
* Have you encountered any issues if this wasn't removed?